exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/DrillRoot.java

```diff
@@ -55,44 +59,89 @@ public Viewable getStats() {
   @GET
   @Path("/stats.json")
   @Produces(MediaType.APPLICATION_JSON)
-  public List<Stat> getStatsJSON() {
-    List<Stat> stats = Lists.newLinkedList();
-    stats.add(new Stat("Number of Drill Bits", work.getContext().getBits().size()));
-    int number = 0;
-    for (CoordinationProtos.DrillbitEndpoint bit : work.getContext().getBits()) {
-      String initialized = bit.isInitialized() ? " initialized" : " not initialized";
-      stats.add(new Stat("Bit #" + number, bit.getAddress() + initialized));
-      ++number;
+  public Stats getStatsJSON() {
+    String version = work.getContext().getOptionManager().getOption(ExecConstants.CLUSTER_VERSION).string_val;
```

Comment:
The assumption for system options is that default values are same across drillbits, and so they need not be put in the persistent store.

In this case, the default values across drillbits might not be the same. This option has a default value of what is returned by `DrillVersionInfo.getVersion()`, and default values are **not put** in persistent store. But if an admin modifies the option, the value is put in the store.